src/composer.json

```diff
@@ -15,7 +15,8 @@
         "symfony/http-foundation": "2.6.*",
         "symfony/dependency-injection": "2.6.*",
         "twig/twig": "1.*",
-        "twig/extensions": "1.*"
+        "twig/extensions": "1.*",
+        "jumbojett/openid-connect-php": "0.1.*"
```

Comment:
This is licensed under Apache2.0 which is incompatible with our GPLv2. That might be a problem for distribution! (ping @mcjaeger)